### ISSUE TITLE
react-window-infinite-loader - React 18 ?

### ISSUE THREAD

**Issue by jy95** [bvaughn/react-window#648]
Hello,

I saw that https://github.com/bvaughn/react-window-infinite-loader is still in React 17 but not yet in React 18.
Can you fix it in `peerDependencies` ?
```js
  "peerDependencies": {
    "react": "^15.3.0 || ^16.0.0-alpha || ^17.0.0 || ^18.0.0",
    "react-dom": "^15.3.0 || ^16.0.0-alpha || ^17.0.0 || ^18.0.0"
  },
```

Thanks in advance

**Comment by vromaniv-liaison** [comment]
+1. I am interested in resolving it as well.

**Comment by jy95** [comment]
@bvaughn released the 1.0.8 version where it is resolved. Thanks ;)
